cli/src/build/onboarding/ui/steps/ios-ci.tsx

```
@@ -0,0 +1,299 @@
+// (the label, the dim reason `message`, and the command lines) separated by a
+// marginBottom={1}, the "Run this in another terminal…" dim note, another
+// <Newline/>, then the Select. Dense: the blank lines + the per-provider reason
+// message drop so two providers' worth of label + command lines still fits 13
+// rows at 60 cols.
+export interface CiSecretsSetupStepProps {
+  advice: CiSecretSetupAdvice[]
+  dense?: boolean
+  onChange: (value: string) => void
+}
+
+export const CiSecretsSetupStep: FC<CiSecretsSetupStepProps> = ({ advice, dense = false, onChange }) => {
```

ESLint

[error] 'dense' is assigned a value but never used. Allowed unused args must match /^_/u.

(unused-imports/no-unused-vars)